tools/pd-ctl/pdctl/command/keyspace_group_command.go

```diff
@@ -47,6 +50,33 @@ func newSplitKeyspaceGroupCommand() *cobra.Command {
 	return r
 }
 
+func newFinishSplitKeyspaceGroupCommand() *cobra.Command {
+	r := &cobra.Command{
+		Use:   "finish-split <keyspace_group_id>",
+		Short: "finish split the keyspace group with the given ID",
+		Run:   finishSplitKeyspaceGroupCommandFunc,
+	}
+	return r
+}
+
+func newMergeKeyspaceGroupCommand() *cobra.Command {
+	r := &cobra.Command{
+		Use:   "merge <target_keyspace_group_id> [<keyspace_group_id>]",
+		Short: "merge the keyspace group with the given IDs into the target one",
```

Comment:
```suggestion
		Short: "merge the keyspace group with the given ID into the target one",
```

Reply:
We do support merge serval groups into one.